### ISSUE TITLE
GLibC Issue

### ISSUE THREAD

**Issue by c3rb3ru5d3d53c** [c3rb3ru5d3d53c/binlex#137]
**Description:**
GLibC Linking Issue

**To Reproduce:**
Get artifacts from CI/CD

**Expected Behavior:**
Works

**Affected OS/Version:**
Linux

**Additional context:**
`/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29'`

**Comment by c3rb3ru5d3d53c** [comment]
https://www.linuxfixes.com/2022/01/solved-where-can-i-find-glibcxx3429.html?m=0

**Comment by mrexodia** [comment]
This is a general problem with Linux unfortunately. You need to build with an older toolchain that what you run on your OS. In this case removing the gcc-11 install might work because you’re fine with c++11

**Comment by c3rb3ru5d3d53c** [comment]
Yeah, I'm in the process of considering that and  doing that in the build logic, :smile: 

Hopefully I'll get to it this this week :smile:

**Comment by c3rb3ru5d3d53c** [comment]
Change settings for Linux build and this works now in the pipeline, still sorting out the whl build
